src/protocol/jdbc/org/apache/jmeter/protocol/jdbc/processor/JDBCPreProcessorResources.properties

```diff
@@ -34,3 +34,5 @@ resultVariable.displayName=Result variable name
 resultVariable.shortDescription=Name of the JMeter variable that stores the result set objects in a list of maps for looking up results by column name.
 queryTimeout.displayName=Query timeout
 queryTimeout.shortDescription=The timeout of statement measured in seconds
+resultSetMaxRows.displayName=Limit ResultSet
```

Comment:
Can you also provide french translations to avoid test failures, (use deepl.com for example)

Reply:
done